.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
+*.{cmd,[cC][mM][dD]} text eol=crlf
+*.{bat,[bB][aA][tT]} text eol=crlf
```

Comment:
i would prefer LF

ref: https://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types

Reply:
Why?

It's written that only `cmd` and `bat` files will have `crlf` line ending. Everything else will have `lf` line ending.

`cmd` and `bat` are included for compatibility purposes (specifically for Windows) if there ever will be the need.

Reply:
Oh, i just noticed cmd & bat only files. Then it's good.